src/styles/blocks/product.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  background: $c-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 47px;
+  }
+
+  &__img {
+    align-self: center;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title-text {
+    color: $c-dark;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+    font-size: 12px;
+  }
+
+  &__product-code {
+    margin: 0;
+    color: $c-ligth;
+    text-align: left;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rate {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__rate-review {
+    margin: 0;
+    color: $c-dark;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 0 16px;
+  }
+
+  &__price-title {
+    margin: 0;
+    color: $c-ligth;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button-buy {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    background-color: $c-header-buttons;
+    border-radius: 5px;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: $c-white;
+    border: $c-header-buttons hidden 1px;
+  }
+
+  &__button-buy:hover {
+    box-sizing: border-box;
+    color: $c-header-buttons;
+    background-color: $c-white;
+    border: $c-header-buttons solid 1px;
```

Comment:
you already set this value without hover, it is no need repeat it, the same with box-sizing

Reply:
haven't fixed